src/__tests__/behaviors/focusZone.tsx

```diff
@@ -7,6 +7,9 @@ async function nextTick() {
   return new Promise(resolve => setTimeout(resolve, 0))
 }
 
+const moveDown = () => userEvent.type(document.activeElement!, '{arrowdown}')
```

Comment:
Verified the magic string `'{arrowdown}'`: https://github.com/testing-library/user-event#special-characters 👍